src/index.html

```diff
@@ -1,19 +1,151 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
           user-scalable=no,
           initial-scale=1.0,
           maximum-scale=1.0,
           minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="info">
+          <label>
+            Surname:
+            <input type="text" name="Surname" autocomplete="off" required>
+          </label>
+        </div>
+        <div class="info">
+          <label>
+            Name:
+            <input type="text" name="Name" autocomplete="off" required>
+          </label>
+        </div>
+        <div class="info">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              data-qa="field-value"
+              min="10" max="100">
+          </label>
+        </div>
+        <div class="info">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date of birth"
```

Comment:
better to write without whitespaces
```suggestion
              name="date_of_birth"
```